src/components/Add-Books.jsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { AddBook } from '../redux/features/books/booksSlice';
 
-const AddBooks = () => (
-  <div className="form-container">
-    <p>ADD NEW BOOK</p>
-    <form>
-      <input type="text" placeholder="Book title" className="input-add-title" />
-      <input type="text" placeholder="Author" className="input-add-author" />
-      <button type="submit" className="btn-add-book">ADD BOOK</button>
-    </form>
-  </div>
-);
+const AddBooks = () => {
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    const id = uuidv4();
+    const author = e.target.elements.author.value;
+    const title = e.target.elements.title.value;
+    const book = {
+      item_id: id, title, author, category: 'Fiction',
+    };
+    dispatch(AddBook(book));
+  };
```

Comment:
- [ ] [OPTIONAL] I would suggest to add a simple check to make sure the title and author values are not empty before you dispatch the addBook action, since currently it makes it possible to just add emty books :+1: